include/dasc_robot/DASCRobots.hpp

```diff
@@ -86,6 +88,8 @@ class DASCRobot : public DASC {
         void emergencyStop();
         bool takeoff();
         bool land();
+        uint64_t get_current_timestamp(); // DEV: in what units? DEV: i thinks ms
```

Comment:
yeah, this is in milli seoconds

Reply:
Nope. Based on the experiments it's in us. 

@chenrc98 please confirm.